tests/core/test_external_search.py

```diff
@@ -1304,6 +1300,16 @@ def pages(worklist):
         # Case-insensitive genre search, genre is saved as 'Fantasy'
         expect([data.lincoln_vampire], "fantasy")
 
+    def test_remove_work(self, end_to_end_search_fixture: EndToEndSearchFixture):
+        search = end_to_end_search_fixture.external_search.search
+        data = self._populate_works(end_to_end_search_fixture)
+        end_to_end_search_fixture.populate_search_index()
+        search.remove_work(data.moby_dick)
+        search.remove_work(data.moby_duck)
+        # Immediately querying never works, the search index needs a second to refresh its cache/index/data
+        time.sleep(1)
+        end_to_end_search_fixture.expect_results([], "Moby")
```

Comment:
Did you mean to add this test? Or did it just come in by accident when rebasing these changes?

Reply:
This is added by me in September, this test was added because I had to make changes to the `doc_type` for `ExternalSearch.remove_work`